clang/test/Driver/sycl-intel-gpu.cpp

```diff
@@ -0,0 +1,118 @@
+/// Tests the behaviors of using -fsycl-targets=intel_gpu*
+
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_bdw -### %s 2>&1 | \
```

Comment:
It needs tests with `-ccc-print-phases` to ensure actions are setup correctly.

Reply:
phases test added